docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,104 @@
+
+Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push // Pushes your changes to your forked branch.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase master // Tacks your commits onto the end of master.
+```
+
+Resolve any conflicts in your editor.
+
+`git push` // Pushes your changes to your forked branch.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn build` - build everything. It is good the run this weekly, or anytime you start a new project off master.
+- `yarn buildto package-name` - build up to a package.
+  - `yarn buildto @fluentui/react`
+
+You can also `cd` to any package under `packages/*` and run `yarn build` to build individual things, though keep in mind that this may require dependencies to be built first (using `build --to`).
+
+### Making a pull request
+
+Make sure all your changes are committed, and run a build on the package that was updated, e.g. `yarn workspace @fluentui/react-components build` for changes to the v9 components. This will compare your changes and detect if the publicly facing API has changed, and update the right docs accordingly. Commit this change.
+
+If your changes make any changes or additions to the DOM, you may need to run `yarn update-snapshots`. Check these updates in.
+
+Before creating a pull request, be sure to run `yarn change` and provide a high-level description of your change, which will be used in the release notes. We follow [semantic versioning](https://semver.org/), so use the guide when selecting a change type:
+
+- Major - Don't do these without expressed agreement from the team. This is a very stable repo. This option is usually disabled.
+- Minor - Adding new API surface area that is backwards compatible and does not dramatically change the intent of an API.
+- Patch - No change in API surface area.
+
+After choosing a change type, the description should follow the [semantic commit message format](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716), and be prepended by one of the following: `feat:`, `fix:`, `chore:`, `docs:`, or `refactor:`.
```

Comment:
can we point to official docs instead of some gist that can get out of date anytime ? https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
Great suggestion! - added the link